src/phrases/phrases-nn.js

```diff
@@ -953,11 +954,24 @@ const phrases = {
     },
     relatedConcepts: {
       placeholder: 'Søk på tittel',
-      articlesTitle: 'Relaterte forklaringar',
-      articlesSubtitle: 'Forklaringar',
+      articlesTitle: 'Forklaringar',
       changeOrder: 'Endre rekkefølgje',
       removeArticle: 'Fjern forklaring',
     },
+    relatedContent: {
+      placeholder: 'Søk på tittel',
+      articlesTitle: 'Innhald',
+      changeOrder: 'Endre rekkefølgje',
+      removeArticle: 'Fjern innhald',
+      addExternal: 'Legg til ekstern artikkel',
+      link: {
+        missingTitle: 'Mangler tittel',
+        missingUrl: 'Url må begynne med http:// eller https://',
+        titlePlaceholder: 'Tittel',
+        urlPlaceholder: 'http://example.com',
```

Comment:
Pirk. Det aller meste av adresser er vel https no.
```suggestion
        urlPlaceholder: 'https://example.com',
```